drift/lib/src/web/web_db.dart

```diff
@@ -166,7 +175,7 @@ class _WebDelegate extends DatabaseDelegate {
 
     while (stmt.step()) {
       columnNames ??= stmt.columnNames();
-      rows.add(stmt.currentRow());
+      rows.add(stmt.currentRow(useBigInt));
```

Comment:
I assume this applies to all integer values then? There's no way to only expose certain columns as `BigInt`? If so, should we adapt the `IntType` implementation to map `BigInt`s from SQL into Dart ints to handle this case?

Also, for the custom wasm implementation in `package:sqlite3`, there might be a case where we _want_ an `int` in Dart (because the user didn't write `int64()`), but actually get a `BigInt` (because that implementation always reports large ints as `BigInt`). For that we also need a conversion in `IntType` I believe?

Reply:
I didn't fully understand. When `useBigInt` enabled (it's only for WebDatabase sql.js implementation) all `int`s read as `BigInt`. At the end doesn't matter it is `int` or `BigInt` because it's always converted to String and mapped back to `int`/`BigInt` depending on column type (this is default Drift behaviour). WASM is returns `BigInt` only if the number > safeInt. As whole, it doesn't matter Drift runs on WebDatabase (with or without `useBigInt`) or on WASM that returns `int` or `BigInt`.

Reply:
Ah fair point. Yeah the implementation could probably be optimized by doing `if (response is BigInt) return response.toInt()` but it should work already.